yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/SimpleProfile.java

```diff
@@ -89,10 +60,12 @@ public Attribute getAttributeByName(String name) {
     public <T> List<Attribute<T>> getAttributesByName(String name, Class<T> clazz) {
         ensureName(name);
         List<Attribute<T>> matches = new ArrayList<>();
-        for (Attribute<?> entry : protectedAttributes) {
-            if (entry.getName().equals(name)) {
-                Attribute<T> value = castSafely(clazz, entry);
-                matches.add(value);
+        for (Map.Entry<String, List<Attribute<?>>> entry : protectedAttributes.entrySet()) {
```

Comment:
Do we need to iterate here or can be access by key?

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/145/commits/535954712a20d28fcfcc17ce17d87cd8468d7302